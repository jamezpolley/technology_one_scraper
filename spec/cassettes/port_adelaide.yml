---
http_interactions:
- request:
    method: get
    uri: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationSearchResults.aspx?Field=S&Period=L7&f=$P1.ETR.SEARCH.SL7&r=P1.WEBGUEST
    body:
      encoding: US-ASCII
      string: ''
    headers:
      Accept-Encoding:
      - gzip,deflate,identity
      Accept:
      - "*/*"
      User-Agent:
      - Mechanize/2.7.6 Ruby/2.5.1p57 (http://github.com/sparklemotion/mechanize/)
      Accept-Charset:
      - ISO-8859-1,utf-8;q=0.7,*;q=0.7
      Accept-Language:
      - en-us,en;q=0.5
      Host:
      - ecouncil.portenf.sa.gov.au
      Connection:
      - keep-alive
      Keep-Alive:
      - '300'
  response:
    status:
      code: 302
      message: Found
    headers:
      Date:
      - Wed, 26 Jun 2019 06:01:51 GMT
      Server:
      - Microsoft-IIS/8.5
      Cache-Control:
      - no-cache, no-store
      Pragma:
      - no-cache
      Content-Type:
      - text/html; charset=utf-8
      Expires:
      - "-1"
      Location:
      - "/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationSearchResults.aspx?Field=S&Period=L7&r=PAE.P1.WEBGUEST&f=P1.ETR.SEARCH.SL7"
      X-Aspnet-Version:
      - 2.0.50727
      X-Powered-By:
      - ASP.NET
      Content-Length:
      - '267'
      Set-Cookie:
      - ASP.NET_SessionId=dcszzo45jh1n2m55lax1rvu4; path=/; HttpOnly
      Via:
      - 1.1 eCouncil.portenf.sa.gov.au
      Keep-Alive:
      - timeout=5, max=100
      Connection:
      - Keep-Alive
    body:
      encoding: UTF-8
      string: "<html><head><title>Object moved</title></head><body>\r\n<h2>Object
        moved to <a href=\"%2fT1PRWebPROD%2feProperty%2fP1%2feTrack%2feTrackApplicationSearchResults.aspx%3fField%3dS%26Period%3dL7%26r%3dPAE.P1.WEBGUEST%26f%3dP1.ETR.SEARCH.SL7\">here</a>.</h2>\r\n</body></html>\r\n"
    http_version: 
  recorded_at: Tue, 14 May 2019 14:00:00 GMT
- request:
    method: get
    uri: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationSearchResults.aspx?Field=S&Period=L7&f=P1.ETR.SEARCH.SL7&r=PAE.P1.WEBGUEST
    body:
      encoding: US-ASCII
      string: ''
    headers:
      Accept-Encoding:
      - gzip,deflate,identity
      Accept:
      - "*/*"
      User-Agent:
      - Mechanize/2.7.6 Ruby/2.5.1p57 (http://github.com/sparklemotion/mechanize/)
      Accept-Charset:
      - ISO-8859-1,utf-8;q=0.7,*;q=0.7
      Accept-Language:
      - en-us,en;q=0.5
      Cookie:
      - ASP.NET_SessionId=dcszzo45jh1n2m55lax1rvu4
      Host:
      - ecouncil.portenf.sa.gov.au
      Connection:
      - keep-alive
      Keep-Alive:
      - '300'
  response:
    status:
      code: 200
      message: OK
    headers:
      Date:
      - Wed, 26 Jun 2019 06:01:53 GMT
      Server:
      - Microsoft-IIS/8.5
      Cache-Control:
      - no-cache, no-store
      Pragma:
      - no-cache
      Content-Type:
      - text/html; charset=utf-8
      Expires:
      - "-1"
      X-Aspnet-Version:
      - 2.0.50727
      X-Powered-By:
      - ASP.NET
      Via:
      - 1.1 eCouncil.portenf.sa.gov.au
      Vary:
      - Accept-Encoding
      Content-Encoding:
      - gzip
      Content-Length:
      - '5531'
      Keep-Alive:
      - timeout=5, max=100
      Connection:
      - Keep-Alive
    body:
      encoding: ASCII-8BIT
      string: !binary |-
        H4sIAAAAAAAAA91ceXfiOLb/u/uc+Q5qXs1U1XQHMISEVCWZY2OzldnMzpw5ObItjIO3ts2W9+a7vyvZbIEkpCYz6RqqiG1td9FPV/daEn/6+foXsVHoDJsSmoS2dfunn683V4J1ZOo3CS200um7Mjxyidtrm4QYyobeGfl9Zs5vEoOzLn9WcG0Ph6ZqkQTSXCckTniTqEg3RDcgJXV7HZqhRaDZnwpmuELuGDVdP0S8Tixs6gRJztgklo7OUHum2mYYEh3JOAjRJRLxKgB2UusWEHyuLdOZIp9YN4kgXFkkmBASJlC48shNIiTLMKUFQQJNfDK+SVBmgy+p1Bj4CpKG6xoWwZ4ZJDXXpuX+Nsa2aa1uGh5xfm1jJ/iSTafNEFum9tv55u5ic3e5uYNy9ImWovlMUKYfB9vAyNwkCw/E3FHJwtTDyQ2XOY8KMzEiLpPJFPznPe+uMyE2CVJNXkqJZIxnVphk0jyW7kB8aDBFe432nurqK3oNNN/0wt2693iOo9QECnxtTbnNkoLU/e8z4q/OuGQ+mU3appO8D6DXU1GNWxTrf+z6diwlDjyHhEVISCAQfuICZDw3gNaxFpquc5MgHR9rUxANdIZpUptgX5soJADZgiQ0sPxbkXb/Tfsv2Pa+NolvQivyJXvyb0ATySaX7EtCqSu1Oyx1fAMpUkdJtiVeKZSTbfkywdC6ww6VXzfn9GI63mythYmp68RJxOzf3fUqUr/d4TtS1MBewhxbMyiUWkhNcdH9VutIuZpoPNTEabq+uJoWjaotZlp93qgI9FrgKwVxOSyUDcXV7J4tDHquXlYWDTM/H/Xrc72fu8f9xUwrXa30Um82MnMPw3bOUbPD2XDQnffEYqtr6dbQEtqddGs+6gkrNStYmqPkZI5fypwgKL0rsTtddmSuDukjb5jpPch28UErGyntnmu1ut1ZV5RmvXSvoHC9Ypdr2aOd9BZXrXVXuWLHKk55Q2uOhKHEGxK9FkYGlalV4Fs7stnDJsjUApm6o8EkLbSs4qhkpSsSNx+VuqSQrs/VPjdRbaUx7HOW0FK6w0HdLxtXoprhwmE/N20Nqo4OearTKpYsfqn0hHYr23vQS1chHtQnesmaq9PeFA+UuppRrLJhCcMMyJety6OBZZUngjgc9B5Gg3p/2F96o1IxjftXMyycV3qlotkGGqOBIfA8TwROmauZZRcPBGjnaq+8kq3OdUgflUBG44iMC6ko9qrWyKlaWsYwWpniPS51x7zRlYDOA17mRUHJtWrdXqs7rXNaFmj1rYfhQPHUTK496o9WwKO7rifQb98YC4bE80K+PFLylVFfAb3XmhFO9D7I7w3tpYVLDEcFEXAlLvLdbwteKBpDIki5idrvFgv28kFxerNhltI6J6KkWCO7yKllhcrIjVpdhfav0lPaslUvtnpVsV3IdTqiRkRLmGh2MRwNFEuzRhO13LNGQsvnW1KBz1B+XNr/JYoDxteiGOloEePByDeLRmXMt7q1bqYHbdXdiqRAv0GfsHKb/GHtnp/XxOEcrstGW0irK35VN/ML+K5qopSj5YUWLd8iBUlf4b7SJdAnQmtZ7BWrQisddpRusd1KG0SK8kXNttKjQdUbdpeelm2tLlstnof/fDMFnzzt9/izKNC/3WJrKQPOZDqWzFxzaHsW6E0AHCyA55W2Whg9G64ZwN19elkrnKfle35WKywMwGWgl3urUYebjQa9NOgtkIFjvV8NcJ/hsatmQku952Z6fxkIPPQtX9D4Xj0HeLDUdg5wvwxgvDPMaatcedTPWSDDvTyl/CtGrReqPegrNk4zPVOeXpnY7t3rUnE6HAhU1gd5+kj2NvD8Up2C0KdXSrfZkRbyfQtkOl9Augi8pvVB1WoCL/pAWQ37i6BSFDjAnjdMw9zaU+Y405tBPqeWlvqgVaOqLA0yQL/PPQi9qwc88KxC8SqzpkEVX+Ve0nWuDDJMW9mqR59xv6gOWhXadqXAT0XWbbQvmR75Mr9gKcLmD9+yXq2rqP1qVD8tCi1G4Dva6TKe1MjuwBibhBoXjZ+akQu19NUCysE4uFq1MlfTUcuDNLCHZejgqTUVbe6hW67ONZBbL1vMPsKYE9i4o/IZrP3+/pjhl7V7d1ET4VsQhE4LxiV8iveVjPzAw7eyqHWmBsxFF/XOlH7BOqVLAtNbVWoZQya82F2CfdYttd+b6WysdquMKB9u7J7AR/MWm78W+T4/yQtg31l/VYpKDviejviKGOdVYxu1LassG+pK8FR7NNdsLrLjpuBpK2Gi20VPBduslrozOuYlftPGkF/0RMBIUBP5KehtRudCvaDc64N6G+wntWkPYKu9UUEBeyoMRv0q46laqqdVkV9US/pqVAS8Rf3VHfargNniqmdblr7I58F2fuNbRqFQahVb6zoS9Iuj0DloOoS5p2vD/ANjQkvTNlpTmK8XFBcx3Xu9tAD9K1O9PNrQ75SsB7048kYDrVksBLwAtnNUUkCfYAAyrWL5CC21VA3WNlOBeWg0qDVxqa1x+Pdlb2ETp9vWq782f+fsB97BXSzxNzfUl6M+b+S9bNwYcJ9P82RKUl1S+E5DOXBpdnJi3+ZSSmcveCkd0XyegNST6p0eL1dEvlNp1NetHyRv3aZ+eSFf3Y+xKS6+ZXNX5W/iQs7mf22DGyGL6VEqVeC/iffeAynw8swPnRxcg1n10uONbyXHL3mCIafM3xsO1CfmBbty95heZS16li/jdHz/jV0nUXm5YkbPuXt5bIqGbJg+u6aiK7R3ya7cvcrStegZ2ovS8XlznBZ4uFbm0bWaiq/NOD3Or8b531h+ysj3a4VuWHSkyfTBTPly5QH/OiOtua4SU50N97v3p59o9zJVWi7WTcdI3KL9T+R1M887rrR5XleFkEBwZ44eJHZy90p42CA0iiN+AjGy8IkvWw5U7DjEl13DZQ39FGebtrETC8JTExNaCEIbCwcBbZycWSxhJ6Z4FM1UbOAgSEHROxubDi2e9EBaxIIYoO36wNwZi5G+pL3l1xiSP+2xeoSZhgOBFGkTf25qJNjw5O4ln8oe8XzXI364OjuVwS1ngYedNfUJU3WHRq2J24hDtOYF4ikouZFti4StnNe/nJ0hULHpoLOzR12617V75HgWcgnY33bebqEQQ4ReIJaFHHfhY2+3i4HeNd4JmSFiXiwWSRrAEmecDDCEzvMknqU2+tUAb5pp9YkamCFBEQfCLAxdJ3Fbdm1yncK3EfdrIviUujETqQ7XVKBAU2mIKdKMewXIb9o+wMaeIp+QfDyzrD7twV3hd0paVOcFCNh38rfge+qWirbFJPxlXXen2Tq7ocEw9HswJaubhAyxPIXFTYJniegbWX1BcmK/K/eVsY3av9zd6W4TAmwBgupPHxnBD2uCH9YEP/728ePnxC27Z93wwue4VAwUu6COhLlFMYD3Ond3iO73wyObtQfh5/LoEJAcfTsM0CPTdqTybldqFqEj4clisd0sRG9neJ/gx8bzcdEODqYgOEmgiAa6QTS57jZ9M0LMceUeFe9pyZ+u9JSo4EfMmsCY9RQP399sPGe80O6mNoPLdiisNXbne34NWpPNIKTjI83dWapFU0q+O/PWRJ4ylC+SfQZHe+Vm1mnMCWAmSEh0msQldvXB+KXJzEBcW+btxm7CpAIzyV1BkduptQjR+60n32JFqbRGUqq3ErcK+X1GghCNXX9HCxhADISo0zCzThFT9dnU9FyZPwwSMkeR0A2IjyoOfdPIXhq+FxQy3w0FKkGqCYUX4DcoJCDh81j4c+acvtRsKsluW1KafLud7FX6idui6xtuCCYKrRvbAOIoSYUYwBLxT6WmSCVKMFmrdyj+osr/5ZjLPmd9mngFpMN3sz7Z74YcAdZTIlFD14+FOA0Ew6QoCRAVxkYoamGthyfRxqjBEIW5zyC05CtpVupFpVIvSRHR3YaeJ71e0fHwSl0tiJp0sEpXc6h32ibazCeCaVlxE6kgxH74NxhDd6D6Qv+CsaFCgTsW1io4hJjhll0OyP53gv/8OfDvrNO82wA4/+4B0J64ngcoKkCf7z2cgskmP0wWeKUTmd11bUSrPz0GogXLVHTZUd5Jg6AtJNtdodbhm82YKGsGQT/uNPVucDyhV1/rwD7X5mtSn8l+isRprr1CLMyA50yDtZv/L3j5+2QY7BXTmISUHHXY78LA2ye5iVijAtZ6HXM3Yz1m45qIVV3Lfc0gsgfU71yC7fSfXYL98GgNtrMBMOWpMzED1CdkugvfN2BKfh1T8h5TbFPBv4GpTu11mqodaKoGUejkrVX1Oq7k2oGq/i1cPb+wf8DV5QFX0a6Qt2aLO38dX9z5AWPc+b+Fs0z+dZxl8gecZfJvxJn4CgOxz5dI7YNI5nRq8905toK3tBHiK2zEI77kA77e0EyIrzATj/VVO6qvNxqT4issxWOFHTD2lsZCfIWxeMzY5WPG3tZgiK8xGI95o/biiNbezGaIr7EZj5mjJuMIc0fMRuSBPvZIiq4bUo/kL44aeF83rshmCfN09+21qbtOXJlYHu/ovG7T19QvUV3/fZ2nemqs87y7Gb90fvKF8/oV9qZk3/WnEI1p5Cm51isQmxod16Mvv/Hr/dUJ92jnJ/HvJlwTGyRez3piPp5wb+DZv6TZ52rF2xMfBcKxPu60WSA9O5o2K326GXgWXn0xWUx8plquNv0arftx6fSfv7ItkdvQeDfcep4FCLwjiiJ77bDuZkYAGn26FhvidWyTJgfj9lgbbAetjFUaMUT5iBWIOX2p7R5dsz+p8Uw6lb5IZdLcFQpdlLlYP60J7Wom1sZ+ALTTH7HiS76p33mOVZgFoWubAesamsiC/P3ReoiB05r3idcnKruNygG9NmELphTibN2dLlkdGZBP4+lZAqyz9wis1Rk8orsDSBT3Xdzy6a9gngj5Hy2Xn8o71Ip5bxMP+zh0/cfc72S8uKou2V64ShrmOIGwFd4knl5TnxAaHrPbKDFHx9uBaPTdz9oQP845REe0zYItAq+FMCi6kEYsi1pV6IWbRDp69rCus+dsAkXssazX69Dw9Tihg9WZhf2eSRYbyXfMSawEzbUs7AXky/rma7wGfR36+4vDirtYL09fhxMUaC7drgPVElvH4YUl41iID/tCfNgK8SEqd1SIj799bLt++EHBdkWnC827AK7PbHW9gBBObt+VQdnVozfK1JxFK+JxAjOOfwwmRRJRAOVRFnce/xj8FdkaHMz2vK77JAgok+v9GChO+2NwGsPQCelMGewAE/qbJf0x2KTQy2Qpe9SrQwXXBkRqRN9wF436VOjf7gx9h3aDtTf09f8M3ywvw/5y8c6S9Hk6xZ1nLlN03mdcAy/w3ToHm6T2hEoWP1zySHDB3KF2iNpSjW+WG4qE2o1up4zaPMrtVixMfBP8AW9CfCSYc9chm6x1mBHqx3UFkwzxHVDzu6kre0xd+VPUBXMPTFMguE9WSF/AjAQpm+zsFRJ87ARgaPk5QQUZIji6vTPSXj63KYiK2PRtkAPR/VIBYgsnqAlDVg71kxX5zqA7P6bFq1O0uDBtm67eeK5rIezoKMBjAsKP6Ujb1gRlEkfHqKMRJPNKqY3qDWWDxYutNoFBFUaqE4T+jLk/AfrU5j+/WqHvj8zcEZ1m0yfotEcAeDqebLGYQYobkAVI4zMNNjqdRp8fxtrLbrXHg2PvY8sEP7oxC3WwAEg257u4/oMD8WJPaQWY+vLpXCbzIgwjVxkiZApBLn+JPhVnlhU7+Z835fKopk3xilrFCIXCRos7GGxrGGYKqM5e4gAaUduF2c0EUxH8QAC8PAZA7gQAVhzGuoXc8dgEuI3N8MydhZv8hmXOAYmKaxA/QIqOYFqRFFTmFaGhxOrMb0pLQUhnlsKELH4UFOaPoZC7PBWFDH/RToeGY60+o7+2pXrnr5ty3DlqY98x8UtTC0gyJyvs0LgJtT2imTC2g/BHAuHVUVXm30yVaSRYK3AyYUQLSoMXdxSZfUNFvi8gufSxoZw5YSh3Fu7GxSEh1sBHPPR1zgVUxeMxLUTVKPMVBfGdckOOFXm+VaQCs7jJeibydn64aZnjjgLyZUfn+RnmMS4zHHUEAxjdELXFvk6Z70n1A/OImj7AMaBty8TA1o8CyWNxSjZ7AiRL2McGoTtvFxgCFHeMYH5AZAnjkY7OA3ReXaKeqQGGwV7CXCNUlG9SvSzxYqzKrb2smjbqzHyHvhX5YfCYPerxpN8YjxzHgxNjw3xNwpdmHUVG9FUEdXpm8f60HwOSx6KY7Pl3edw0erZpqkExV1AaQ7FRR01e+RbpK70zgAUSUsdcmJlWiCqOTn1wk/xAkzSXO4pB7o0xWLRQBvFeiFhkicqmo9vYDyZRUFgplTvtJ9wg3wy1iQXTU+SQ80HgwgxOnYLNxE5e4Zq/M0wvjsE0dwJMRWJDHMJUDlZzbSiZ8jGoHa/VAkE4i55n/g4Kc+eoj1fMXa/x9XqlXirztVjX29iHV01XnzlIxHMT/Hpj5jjmyjxZs/Tw4/5bcx1pLn3179wkLhK38drA9nX/+vQStLE9yESJseUYbrPWFdP/T70+pGuwHzK0e/ZDz/8offbmMvt+9M8p/fP3o5+j9HPvR/+C0r94P/qXlP7l+9HPU/rbSG09UlOboQq3dDjfPrIHbBEwztpdPzzYRvDcmuppT+zU/mbzy/rftszuHo/od5Ki856BTMZ7m0L2zuMHbNvF+jz+vkbV0AGHfW66s2BzAH+bcLh4uVd+53Rq8+jp1OZm8ZcyyPu+u1gfUF0raFfcw+Xpk09mPqf7fyXvNdt5jqZ5wK53kLiWyHTmECvRH0A7VmzvlD6ozaZHtHf2+DyztfxoXRaQHfxMwJM7pHbPJk+8eJdFtGOL7hzn9gDQOwqA3u7PqT1zNjzEvkHCm8RdQKwxuGzROW8UH/R+ejdSNIH/XzyP03PHB8f8I4Y3uw62x0Lm4BeaGraYUl/citDblhYsME3HdiXEmxC47S4EDu6Onv9/5nj+C2rP7Km9c1TtnZPU7vnmHGurA+03o/S31HrmR9d6dk/r3aNa756kdfqrfmC7j2CepaNu8MzJ+1P3vj1pto6ZhYLrrXyqw8TtXzS4/4o6RJs4ruUaK9RwIJgxYf4ANzzy3k8zfMd/fIRNcM/+uiAUTKWuf/l7QeQ7/N//9POHT7qrzejWi89Jn2B99WkMtoEFDZ8+o/81x+jTh08f/yfqOJjAJJhk/ODj56QZfPr4JbauHz+jmxsEQQSBKrT43V0bgmBZbjbaFfrjMwOoAJU/fUx//Pz1WIHhboF//vPz11TqH/9gW1Xj3zikt/TUMbtZ/45iKv5ZzP8HddY2eTJTAAA=
    http_version: 
  recorded_at: Tue, 14 May 2019 14:00:00 GMT
recorded_with: VCR 5.0.0
