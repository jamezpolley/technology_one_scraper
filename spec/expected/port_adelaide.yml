---
- council_reference: 040/1427/19
  address: 7A Boord St SEMAPHORE SOUTH SA 5019
  description: Shed
  info_url: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=040%2F1427%2F19&f=%24P1.ETR.APPDET.VIW&r=PAE.P1.WEBGUEST
  date_received: '2019-06-20'
  date_scraped: '2019-05-15'
- council_reference: 040/1428/19
  address: 39 Branson Ave CLEARVIEW SA 5085
  description: Single storey dwelling
  info_url: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=040%2F1428%2F19&f=%24P1.ETR.APPDET.VIW&r=PAE.P1.WEBGUEST
  date_received: '2019-06-20'
  date_scraped: '2019-05-15'
- council_reference: 040/1429/19
  address: 2 Brenda Tce LARGS NORTH SA 5016
  description: Swimming pool and safety fence
  info_url: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=040%2F1429%2F19&f=%24P1.ETR.APPDET.VIW&r=PAE.P1.WEBGUEST
  date_received: '2019-06-20'
  date_scraped: '2019-05-15'
- council_reference: 040/1430/19
  address: 32 Rosewater Tce OTTOWAY SA 5013
  description: Verandah
  info_url: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=040%2F1430%2F19&f=%24P1.ETR.APPDET.VIW&r=PAE.P1.WEBGUEST
  date_received: '2019-06-20'
  date_scraped: '2019-05-15'
- council_reference: 040/1431/19
  address: Oliver Rogers Rd OUTER HARBOR SA 5018
  description: Internal office fit-out
  info_url: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=040%2F1431%2F19&f=%24P1.ETR.APPDET.VIW&r=PAE.P1.WEBGUEST
  date_received: '2019-06-20'
  date_scraped: '2019-05-15'
- council_reference: 040/1432/19
  address: 4B Jaffrey St BLAIR ATHOL SA 5084
  description: Two storey detached dwelling
  info_url: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=040%2F1432%2F19&f=%24P1.ETR.APPDET.VIW&r=PAE.P1.WEBGUEST
  date_received: '2019-06-20'
  date_scraped: '2019-05-15'
- council_reference: 040/1433/19
  address: 97 Victoria Rd BIRKENHEAD SA 5015
  description: Garage forward of the existing dwelling
  info_url: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=040%2F1433%2F19&f=%24P1.ETR.APPDET.VIW&r=PAE.P1.WEBGUEST
  date_received: '2019-06-20'
  date_scraped: '2019-05-15'
- council_reference: 040/1434/19
  address: 3 Boomerang Rd CROYDON PARK SA 5008
  description: Verandah
  info_url: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=040%2F1434%2F19&f=%24P1.ETR.APPDET.VIW&r=PAE.P1.WEBGUEST
  date_received: '2019-06-20'
  date_scraped: '2019-05-15'
- council_reference: 040/1435/19
  address: 54 Ways Rd MANNINGHAM SA 5086
  description: Demolition of dwelling and all associated structures
  info_url: https://ecouncil.portenf.sa.gov.au/T1PRWebPROD/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=040%2F1435%2F19&f=%24P1.ETR.APPDET.VIW&r=PAE.P1.WEBGUEST
  date_received: '2019-06-20'
  date_scraped: '2019-05-15'
